File: poo/java/Quadrado/Quadrado.java
/*
 *  Programa escrito com propósito didático apenas.
 *  @author Nathan Cruz <nathan.cruz@unifesp.br>
 *  @created 25/03/14
 *  R.A.: 86492
 */

public class Quadrado{

    /* Atributos */
    private float lado;

    /* Construtores */
    public Quadrado(){
        this.lado = 10;
    }
    public Quadrado(float lado){
        this.lado = lado;
    }

    /* Setters */
    public void setLado(float lado){
        this.lado = lado;
    }

    /* Getters */
    public float getLado(){
        return this.lado;
    }

    /* Calcula a area */
    public float Area(){
        return (this.lado)*(this.lado);
    }

    /* Aumenta o tamanho do quadrado */
    public void Aumenta(float val){
        this.setLado(this.getLado()+val);
    }
}

File: poo/python/Quadrado/Quadrado.py
#!/usr/bin/env python
#-*- coding: utf-8 -*-

"""
@author Nathan Cruz <nathan.cruz@unifesp.br>
@created 27/03/14
"""
from __future__ import unicode_literals
from __future__ import print_function
import random

class Quadrado:

    # Atributos #
    __lado = None

    # Construtor #
    def __init__(self, lado=10):
        self.__lado = lado

    # Setters #
    def setLado(self, lado):
        self.__lado = lado

    # Getters #
    def getLado(self):
        return self.__lado

    # Calcula a area #
    def Area(self):
        return self.__lado * self.__lado

    # Aumenta o quadrado #
    def Aumenta(self, val):
        self.setLado(self.getLado()+val)

if __name__ == '__main__':

    lista = []
    soma = 0
    for i in range(10):
        lista.append(Quadrado(random.random()*10))
        soma += lista[i].Area()

    print('A soma eh: %.2f' % (soma))    
